### ISSUE TITLE
HRH Analysis - Malawi TDY Prep - Vamsi

### ISSUE THREAD

**Issue by jb3436** [ICPI/Engagements#30]
Malawi TDY Prep HRH Analysis 
--
Request Received: August 1, 2018
Discussed at Leads Monday Aug 6,  2018

**Request**
There is a TDY to Malawi in September to review HRH allocation and alignment to program targets and performance. During RPM Vamsi requested the country team conduct analysis on their HRH data looking at changes in clinical cascade indicators. They did a decent job, but there is more we could do in order to understand the impact of surge hiring in Malawi. A few points to note:
- The HRH data is from the Malawi “audit”. The audit is similar to the HRH inventory from Tanzania and HRID in South Africa, but it captures only number of workers. There is no salary info. 
- The country team chose to use the audit data instead of HRH_CURR because they classified their workers by ART providers and HIV Diagnostic Assistants (HDA)  who are responsible for testing and linkage. We want to use the audit data (attached) for this analysis rather than HRH_CURR.

**Analysis request:** 
We think this is a good start, but we would like to see some more analysis. The overall goal is to understand if surge hiring (HRH difference between FY16 and FY17) led to improved performance at site-level. Performance can be defined as achievement against TX_CURR targets, and/or a change in TX_NET_NEW from Fy16 to FY17. Please see below for what we are looking for and advise if it is possible:
- Site level total HRH and TX_CURR achievement (targets vs. results) for FY16 and FY17.
- Site level changes in ART providers and TX_CURR achievement for FY16 and FY17.
- Site level changes in HDAs and HTS_TST_POS achievement for FY16 and FY17.
- Site level changes in total HRH and TX_NET_NEW for FY16 and FY17.
- We would like the above analysis presented by scale-up vs. saturation districts, if possible. 
- Some way of visualizing ART providers by patient volume (perhaps a bubble chart?)
- We are open to other ways to looking at the data so please feel free to suggest other ideas.

**Due Date:**
Draft analysis by end of August so we can make any necessary changes before our TDY (September 17-21). 

**Final Due date:**
Mon September 10, 2018

**Next steps:**

- [x] meet to discuss timeline
- [x] complete analysis and do QC
- [x] include source, caveats, branding, and acknowledgements, and findings
- [x] send to Vamsi for reveiw
- [ ] request follow up on how analysis was used
- [ ] send to leads as FYI
- [ ] schedule at brownbag? 

Resources
--
Project folder [here on SharePoint](https://www.pepfar.net/OGAC-HQ/icpi/Shared%20Documents/Forms/AllItems.aspx?RootFolder=%2FOGAC%2DHQ%2Ficpi%2FShared%20Documents%2FClusters%2FHSS%2FProjects%2FMalawi%20HRH%20Analysis&FolderCTID=0x01200080CC6F83D1766F4D9E3497D4529EC74A&View=%7B94C838B2%2DE166%2D4122%2DB8B4%2D7BEB9E1BC12B%7D)

**Comment by kkale25** [comment]
Yaa committed to taking a first stab at the analysis with a goal to have it out to the cluster for QC and review by Tuesday. Kashmira to help make changes/updates as needed.

**Comment by jb3436** [comment]
/remind @jb3436 to follow up with Vamsi on October 1 about how this analysis was used

**Comment by jb3436** [comment]
Final product sent to Vamsi on 9/13. 
Subject: Malawi.xls

Vamsi requested that HTS POS be included in the analysis and Jasmine incorporated the data
Kashmira conducted a QC of the data sent by the country team and it did not match other MER values
Travis replaced the country submitted data with data from the MSD
Analysis includes a summary of methods, data source, acknowledgements and summary purpose and objectives for the visualizations

Will close out ticket once we hear how the analysis was used in country

**Comment by Yabayabadoo** [comment]
Tool updated to fix issues and re-submitted

**Comment by Yabayabadoo** [comment]
Additional Analytic request
Is it possible to further separate these clusters in to more detail? For example, it would be helpful to see where most of the sites are clustered for expert clients – are we seeing more sites in 0-5 vs. 6-10 increase in expert clients? Similarly, are we seeing more sites in 0-2 vs. 3-5 increase in ART providers?

**Comment by jb3436** [comment]
10/5 - followed up with Vamsi on how the analysis was used

**Comment by jb3436** [comment]
How Analysis was Used
--
Response from Vamsi: 

**Summary of how the analysis was used:**
- Vamsi and Cate McKinney (the HRH iSME for Malawi) visited Malawi to review the HRH analysis and conduct site visits to assess the impact of HRH surge hiring.
- The analysis helped us identify sites where the surge hiring did/ did not lead to increase in select indicators. 
- We reviewed select sites with the country team and implementing partners to understand the impact of HRH surge hiring.
- We initiated the discussion with IPs and country team on “right sizing” HRH allocation across sites. The analysis provided us a good baseline to initiate the discussion. Right sizing involves allocation and redistribution of HRH across sites. 
- The country team will do a deep dive into select sites that were identified from the analysis .The goal is determine factors outside HRH that might be contributing to better performance.
- The Malawi team found the analysis to be useful and they plan to repeat the analysis using FY18 data.

FYI @kkale25 @Yabayabadoo @wvp3
